corehq/apps/app_manager/app_schemas/case_properties.py

```diff
@@ -272,11 +272,13 @@ class ParentCasePropertyBuilder(object):
     Full functionality is documented in the individual methods.
 
     """
-    def __init__(self, app, defaults=(), per_type_defaults=None, include_parent_properties=True):
+    def __init__(self, app, defaults=(), per_type_defaults=None, include_parent_properties=True,
+            validate_properties=False):
         self.app = app
         self.defaults = defaults
         self.per_type_defaults = per_type_defaults or {}
         self.include_parent_properties = include_parent_properties
+        self.validate_properties = validate_properties
```

Comment:
Maybe this is just me so feel free to ignore but I tend to think of validation as raising an exception or something similar. I might rename this `exclude_invalid_properties`

Reply:
good point, updated